src/query/settings/src/settings_default.rs

```diff
@@ -624,6 +624,12 @@ impl DefaultSettings {
                     mode: SettingMode::Both,
                     range: Some(SettingRange::Numeric(0..=1)),
                 }),
+                ("disable_variant_check", DefaultSettingValue {
+                    value: UserSettingValue::UInt64(0),
+                    desc: "Disable variant check to allow insert invalid JSON values",
```

Comment:
If the invalid JSON insert, what will happen to query them?

Reply:
jsonb's functions will check whether the JSON is valid or not, if it is not a valid JSON, the function will return the default value, so do not worry about the runtime will report an error.